_includes/context/priority.md

```diff
@@ -1,3 +1,97 @@
+- Only one context _form_ can be shown for a selection or cursor position, they cannot be concatenated to another context _form_ or a context _toolbar_.
+- Context _toolbars_ with `position: 'selection'` are prioritized over `position: 'node'`, and `position: 'line'` is given the lowest priority.
+- When the editor concatenate context _toolbars_ where there is more than one context _toolbar_ to display with:
+  
+    - A matching predicate,
+    - The same `scope` and `position` values.
+
+  Concatenated toolbars may contain duplicate toolbar items.
+
+- If no matching context toolbars or context forms are found for the selection or cursor position, then editor will recursively search for matches on the parent node of the current node, until it reaches the root node of the editor content.
+
+##### Description of how context toolbars and context forms are prioritized
+
+The following description can be used for troubleshooting the behavior of context toolbars and context forms.
+
+The editor will determine which context toolbars or context form will be shown using following process:
+
+1. Find all **context forms** with both:
+
+    - `scope: 'node'`
+    - A predicate matching the current selection or cursor position.
+
+    If there any matching **context forms**, the first one found will be displayed in the editor and the process will end.
+
+2. Find all **context forms** with both:
+
+    - `scope: 'editor'`
+    - A predicate matching the current selection or cursor position.
+
+    If there any matching **context forms**, the first one found will be displayed in the editor and the process will end.
+
+3. Find all **context toolbars** with both:
+
+    - Any `scope`
+    - A predicate matching the current selection or cursor position.
+
+    If there any matching **context toolbars**, the editor will prioritize the context toolbars based on the `position` value.
+
+    - All context toolbars with `position: 'selection'` or `position: 'node'` will be concatenated, the concatenated toolbar will be displayed, and the process will end.
+    - Otherwise, all context toolbars with `position: line` will be concatenated, the concatenated toolbar will be displayed, and the process will end.
+
+4. Find all **context forms** with both:
+
+    - Any `scope`
+    - A predicate matching the **parent node** of the current node, selection, or cursor position.
+
+    If there any **context forms** found, the first one found will be displayed in the editor and the process will end.
+
+5. Find all **context toolbars** with both:
+
+    - Any `scope`
+    - A predicate matching the **parent node** of the current node, selection, or cursor position.
+
+    If there any matching **context toolbars**, the editor will prioritize the context toolbars based on the `position` value.
+
+    - All context toolbars with `position: 'selection'` or `position: 'node'` will be concatenated, the concatenated toolbar will be displayed, and the process will end.
```

Comment:
This is the point at which it won't combine position selection and node toolbars - selection first, then if none found try node, then if none found try line. Different to step 3.

Reply:
Ah, copied but forgot to adapt.